content/en/flux/installation/_index.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Installation"
+description: "Flux install, bootstrap, upgrade and uninstall documentation."
+weight: 30
+---
+
+This guide walks you through setting up Flux to
+manage one or more Kubernetes clusters.
```

Comment:
This is no longer a guide style doc. Can you please write here an introduction and explain what Flux is made of, and give a quick overview of what people should expect from this doc.